include/glow/Graph/Graph.h

```diff
@@ -314,6 +314,9 @@ class Function final : public Named {
   ReshapeNode *createExpandDims(llvm::StringRef name, NodeValue input,
                                 llvm::ArrayRef<size_t> axes);
 
+  ReshapeNode *createFlatten(llvm::StringRef name, NodeValue input,
```

Comment:
Please add a doxygen line that describes the semantics of the operator and the parameter.